cmd/oci-runtime-tool/generate.go

```diff
@@ -58,9 +58,9 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "poststop", Usage: "set command to run in poststop hooks"},
 	cli.StringSliceFlag{Name: "prestart", Usage: "set command to run in prestart hooks"},
 	cli.BoolFlag{Name: "privileged", Usage: "enable privileged container settings"},
-	cli.BoolFlag{Name: "read-only", Usage: "make the container's rootfs read-only"},
-	cli.StringFlag{Name: "root-propagation", Usage: "mount propagation for root"},
-	cli.StringFlag{Name: "rootfs", Value: "rootfs", Usage: "path to the rootfs"},
+	cli.StringFlag{Name: "rootfs-path", Value: "rootfs", Usage: "path to the rootfs"},
+	cli.StringFlag{Name: "rootfs-propagation", Usage: "mount propagation for rootfs"},
+	cli.BoolFlag{Name: "rootfs-readonly", Usage: "make the container's rootfs readonly"},
```

Comment:
The config.json property is `root`, so I'd prefer sticking to `root` instead of switching to `rootfs`.  Other than that these changes look good to me.